packages/babel-preset-react-app/create.js

```diff
@@ -172,6 +177,10 @@ module.exports = function(api, opts, env) {
         require('babel-plugin-dynamic-import-node'),
     ].filter(Boolean),
     overrides: [
+      isFlowEnabled && {
+        test: /\.(js|mjs|jsx)$/,
```

Comment:
How about using `exclude` instead of `test` and feeding it `/\.tsx?$/`?

Reply:
I like it. Updated